src/sentry/api/serializers/models/team.py

```diff
@@ -154,7 +158,14 @@ def get_attrs(
         result: MutableMapping[Team, MutableMapping[str, Any]] = {}
 
         for team in item_list:
-            is_member = team.id in memberships
+            membership: OrganizationMemberTeam = memberships.get(team.id)
```

Comment:
again, this tripped me up -- `team_membership` would be preferred.

Reply:
Done.